src/XRP/xrp-utils.ts

```diff
@@ -110,4 +110,5 @@ function xrpToDrops(xrp: BigNumber.Value): string {
     .toString(10)
 }
 
-export { xrpToDrops, dropsToXrp }
+const XRPUtils = { dropsToXrp, xrpToDrops }
+export default XRPUtils
```

Comment:
👍

Reply:
Maybe this is the pattern I should use in `xpring-common-js` for this same problem?

(As opposed to:
```ts
const XRPUtils = {
  dropsToXrp(drops) {...},
  ...
}
```

Reply:
This does seem cleaner to me but I ultimately don't care as long as we're consistent.